statistics-dashboard-worker/src/main/java/eu/europeana/statistics/dashboard/worker/execution/HistoricalDataRunner.java

```diff
@@ -1,57 +1,26 @@
 package eu.europeana.statistics.dashboard.worker.execution;
 
 import eu.europeana.statistics.dashboard.common.internal.model.Historical;
-import eu.europeana.statistics.dashboard.common.internal.MongoStatisticsField;
 import eu.europeana.statistics.dashboard.service.persistence.MongoSDDao;
 import eu.europeana.statistics.dashboard.service.persistence.StatisticsData;
 import eu.europeana.statistics.dashboard.service.persistence.StatisticsQuery;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
-import java.time.LocalDateTime;
 import java.util.List;
 
-
 public class HistoricalDataRunner{
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AnalyzerRunner.class);
 
     public static void runHistoricalScript(MongoSDDao mongoSDDao){
         List<String> countries = mongoSDDao.getAllCountryValuesTargetCollection();
-
         for(String country : countries) {
-            writeHistoricalData(country,mongoSDDao);
+            writeHistoricalData(country, mongoSDDao);
```

Comment:
Since this method `writeHistoricalData` is a very simple method, you can simply remove said method and write here its code, so it will look something like this:
```
for(String country : countries) {
   Historical result = mongoSDDao.generateHistoricalSnapshot(country);
   mongoSDDao.saveHistoricalRecord(result);
}
```

Based on a different comment, here would be better to write the logs.